src/main/java/edu/gatech/chai/VRDR/messaging/CauseOfDeathCodingMessage.java

```diff
@@ -44,8 +44,8 @@ public void setCauseOfDeathCodedContentBundle(CauseOfDeathCodedContentBundle cau
 
     public CauseOfDeathCodingMessage(BaseMessage messageToCode) {
         this(messageToCode == null ? null : messageToCode.getMessageHeaderId(),
-                messageToCode == null ? null : messageToCode.getMessageDestination(),
```

Comment:
This order should be kept and not flipped.  It confused me at first, but the coding messages are reverse flow back to the jurisdiction.